packages/statemanager/src/flat/snapshot.ts

```diff
@@ -0,0 +1,277 @@
+import { merkleizeList } from '@ethereumjs/trie'
+import {
+  Account,
+  KECCAK256_NULL,
+  KeyEncoding,
+  LevelDB,
+  bigIntToBytes,
+  bytesToBigInt,
+  bytesToHex,
+  equalsBytes,
+  hexToBytes,
+} from '@ethereumjs/util'
+import debugDefault from 'debug'
+import { keccak256 } from 'ethereum-cryptography/keccak'
+
+import type { Address } from '@ethereumjs/util'
+import type { Debugger } from 'debug'
+
+export const ACCOUNT_PREFIX: Uint8Array = hexToBytes('0x' + '00')
+export const STORAGE_PREFIX: Uint8Array = hexToBytes('0x' + '11')
+export const CODE_PREFIX: Uint8Array = hexToBytes('0x' + '22')
+
+const { debug: createDebugLogger } = debugDefault
+
+type SnapshotElement = {
+  data: Uint8Array | undefined
+}
+
+function concatenateUint8Arrays(arrays: Uint8Array[]) {
+  const l = arrays.reduce((prev, curr) => prev + curr.length, 0)
+  const concatenatedArray = new Uint8Array(l)
+
+  let offset = 0
+  for (const array of arrays) {
+    concatenatedArray.set(array, offset)
+    offset += array.length
+  }
+
+  return concatenatedArray
+}
```

Comment:
I believe this functionality is already covered by `concatBytes` from `ethereum-cryptography/utils`

Reply:
Good catch! I've switched over.